newdle/client/src/components/UserMenu.js

```diff
@@ -52,7 +52,7 @@ export default function UserMenu() {
     </Dropdown>
   ) : (
     <div className={styles['unknown-user']} onClick={login}>
-      <span>Log in</span>
+      <a styleName="login-link">Log in</a>
```

Comment:
```suggestion
      <a className={styles['login-link']}>Log in</a>
```

Reply:
My fault! I forgot we were not using it in newdle 😛

Reply:
Indeed. ;)